lib/metered_raven_error_handler.rb

```diff
@@ -8,7 +8,7 @@ def call(ex, context)
 
       retry_count = (context['retry_count'] || (context[:job] && context[:job]['retry_count'])).try(:to_i)
 
-      if retry_count.nil? || retry_count > 1
+      if !retry_count.nil? && retry_count > 0
```

Comment:
If we want to support jobs where retrying has been disabled, we could check the `retry` key.

Reply:
I don't understand this comment.

Reply:
Ha, that's because the first half of this train of thought was in my head.
I was thinking: maybe we were including nil `retry_count`s in here in case there are jobs that don't retry at all?
But if that's the case, it would be more proper to check the `retry` parameter instead of the `retry_count` one.